ShinyExpenseTrackerTests/src/ca/ualberta/cs/shinyexpensetracker/test/AddExpenseClaimActivityTest.java

```diff
@@ -0,0 +1,119 @@
+        public void onDateSet(DatePicker view, int year, int monthOfYear, int dayOfMonth) {
+        	int yr = year;
+            int month = monthOfYear;
+            int day = dayOfMonth;
+        }
+    };
+    
+    protected void setUp() throws Exception {
+    	super.setUp();
+        instrumentation = getInstrumentation();
+        activity = (AddExpenseClaimActivity) getActivity();
+    	
+    	DatePickerDialog datePicker = new DatePickerDialog(instrumentation.getContext(), dateListener, TARGET_YEAR, TARGET_MONTH, TARGET_DAY);
+    	name = ((EditText) activity.findViewById(ca.ualberta.cs.shinyexpensetracker.R.id.editTextExpenseClaimName));
+    	startDate = ((EditText) activity.findViewById(ca.ualberta.cs.shinyexpensetracker.R.id.editTextStartDate));
+    	endDate = ((EditText) activity.findViewById(ca.ualberta.cs.shinyexpensetracker.R.id.editTextEndDate));
+    }
+    
+	public void teststartDate() {
+		instrumentation.runOnMainSync(new Runnable() {
+			public void run(){
+				startDate.performClick();
+				
+				assertTrue("startDate dialog is working", ((AddExpenseClaimActivity) activity).getStartDateDialog().isShowing());
+			}
+		});
+	}
+	
+	public void testendDate() {
+		instrumentation.runOnMainSync(new Runnable() {
+			public void run(){
+				endDate.performClick();
+				
+				assertTrue("endDate dialog is working", ((AddExpenseClaimActivity) activity).getEndDateDialog().isShowing());
+			}
+		});
+	}
+	
+	public void testText() {
+		instrumentation.runOnMainSync(new Runnable() {
+			@Override
+			public void run() {
+				name.setText("URoma Trip");
+				startDate.setText("01-04-2015");
+				endDate.setText("08-04-2015");
+				
+				assertEquals("URoma Trip", name.getText().toString());
+				assertEquals("01-04-2015", startDate.getText().toString());
+				assertEquals("08-04-2015", endDate.getText().toString());
+			}
+		});
+	}
+	
+	public void testExpenseClaim() {
+		instrumentation.runOnMainSync(new Runnable() {
+			@Override
+			public void run() {
+				String name = "URoma Trip";
+				@SuppressWarnings("deprecation")
+				Date startDate = new Date(2015, 01, 01);
+				@SuppressWarnings("deprecation")
+				Date endDate = new Date(2015, 01, 10);
+				Status status = null;
+				Tag tag = null;
+				ExpenseClaim expenseClaim = new ExpenseClaim(name, startDate, endDate, status, tag);
+				ExpenseClaimList claimList = new ExpenseClaimList();
+				claimList.addClaim(expenseClaim);
+				
+				assertTrue("name is not 'URoma Trip'", expenseClaim.getName().toString() == "URoma Trip");
+				assertTrue("startDate is not 'null'", expenseClaim.getStartDate().toString() == "2015-01-01");
+				assertTrue("endDate is not 'null'", expenseClaim.getEndDate().toString() == "2015-01-10");
+				assertTrue("Status is not 'null'", expenseClaim.getStatus() == null);
+				assertTrue("Tag is not 'null'", expenseClaim.getTag() == null);
+				
+				assertTrue("ClaimList is empty", claimList.size() == 1);
```

Comment:
Can you change these assertTrue's to assertEquals? Java's == operator is really dumb with objects.